cloud/infoblox

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+#  Copyright 2014 "Igor Feoktistov" <ifeoktistov@yahoo.com>
+#Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+DOCUMENTATION = '''
+module: infoblox
+version_added: "1.0"
+short_description: Manage infoblox opeartion by useing rest api .   
+There are  4 states to call infoblox module.
```

Comment:
docs do not follow normal structure, please check other modules for how to do this